test/lookup_results.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file Test the lookup results for all points in `./lookup_points.json`
+ *    against the values in their `expected` properties.
+ */
+
+'use strict';
+
+var tape = require( 'tape' );
+var supervisor = require( '../');
+var lookupPoints = require( './lookup_points.json' );
+var util = require( 'util' );
+
+tape( 'Test actual lookup results against expected.', function ( test ){
+  supervisor.lookup( function ( lookup ){
+    var numCompletedTestCases = 0;
+
+    function lookupTestCase( testCase ){
+      lookup.search( testCase.point, function ( result ){
+        for( var key in testCase.expected ){
+          var msg = util.format(
+            '`%s` matches expected for %s.',
```

Comment:
this is verbose. could we output errors per tag and only a single pass message for each lat,lon?